### ISSUE TITLE
xmr-stak 2.10.1 crash regularly

### ISSUE THREAD

**Issue by Theflyer737** [fireice-uk/xmr-stak#2344]
Please provide as much as possible information to reproduce the issue.

# Basic information
  Threadripper 1950x
Nvidia 1060
Nvidia 1070Ti

# Compile issues
 Windows 10 pro
  Compiled as usual with instructions given on your site.
 Always compiled other versions the same way.
# Issue with the execution
  - Do you compiled the miner by our own? Yes (see comment above)
Errors reported in windows events log file:

Nom de l’application défaillante xmr-stak.exe, version : 0.0.0.0, horodatage : 0x5c8a6be9
Nom du module défaillant : ucrtbase.dll, version : 10.0.17763.348, horodatage : 0x26a80883
Code d’exception : 0xc0000409
Décalage d’erreur : 0x000000000006e91e
ID du processus défaillant : 0x1b28
Heure de début de l’application défaillante : 0x01d4db38ea73d593
Chemin d’accès de l’application défaillante : C:\Users\Calypso\xmr-stak-2.10.1\build\bin\Release\xmr-stak.exe
Chemin d’accès du module défaillant: C:\WINDOWS\System32\ucrtbase.dll
ID de rapport : 769bc9b1-1c84-41e9-9e29-ceb9858de912
Nom complet du package défaillant : 
ID de l’application relative au package défaillant : None

# Stability issue
  - Is the CPU or GPU overclocked? slightly and this was also the case with ^previous xmr-stak versions which were stable
  - Is the Main memory of the CPU or GPU undervolted? no

The issue is that xmr-stak crashes regularly but at different timings.
It could run during 5 hours or only for 10 minutes.
This issue didn't existed in previous versions (2.9.xxx and before).
This appears since v2.10.x

**Comment by IceFloe** [comment]
Have you seen the crash message? Did you run it from command line directly? I also have crash in cuda backend, just wonder If it is the same xD

**Comment by psychocrypt** [comment]
please note that 2.10.2 is release wich fixes a bug with opencl (amd)

**Comment by IceFloe** [comment]
He has Nvidia that’s why I think this is CUDA error

**Comment by Theflyer737** [comment]
Hi, thanks for your quick reaction.
IceFloe - There are no crash messages except the one above. I run xmrstak via the .exe file in the \bin directory.
The issue seems to be solved now, I didn't experienced any new crashes since yesterday 07pm.
I did 2 things for this: 
. Update visual studio with the latest updates
. I noticed that I copied the *.dll in \bin directory from 2.9.x version. This was a mistake from me. I copied the correct ones as mentioned on the compile section from github site.
I don't know which of the things solved the issue.
I have another annoying issue: I had an hashrate of 750 with my 1070ti and now I have  610.
Nvidia GPU's seem to be affected by the new release.
I will not go for 2.10.2 if it is only opencl AMD bug fix. I will wait for the next one depending on the bug fixes applied.
